src/components/Menu/Menu.tsx

```diff
@@ -98,7 +98,6 @@ export const Menu = ({
     return (
         <nav
             className={merge([CONTAINER_CLASSES, isMenuOpen ? 'tw-block' : 'tw-hidden'])}
-            role={isMenuOpen ? 'dialog' : ''}
```

Comment:
Not so sure about this one, why the removal?

Reply:
The Nav element wrapping an element with the role "menu" doesn't necessarily need an explicit role specification. 

I addressed it though, and now added an explicit role "navigation"